### PR TITLE
Update community-ecosystem-apps.md

### PR DESCRIPTION
**Pull request by wjakelee**
## Purpose of this pull request

This pull request is to update the FAQ section of this document

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ X] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/community-ecosystem-apps.md [hunk 1]
```diff
@@ -33,9 +33,43 @@ More information on this initiative and how to contribute can be found [here](#f
...
+
+Please follow the guidelines outlined in the [Sumo Logic Community Content Submission](https://forms.gle/KQBLBuMuUw85xtRi9) form if you would like to contribute your own content to this repository and have our team review it for publishing.
+
+**Alternative Submission:**
+Follow the "fork-and-pull" Git workflow:
+- **Fork** the repo on GitHub
+- **Clone** the project to your own machine
+- **Create** any new folders/subfolders necessary for your content
+	- Add your content to these folders
+	- Verify that all sensitive data has been removed BEFORE submitting a PR. This includes PII, Credentials, and Network data. Screenshots included (either blurr or mark over).
```
Comment (kimsauce) [inline docs/integrations/community-ecosystem-apps.md:49]:
```suggestion
	- Verify that all sensitive data has been removed BEFORE submitting a PR. This includes PII, Credentials, and Network data. Screenshots included (either blur or mark over).
```

File: docs/integrations/community-ecosystem-apps.md [hunk 2]
```diff
@@ -33,9 +33,43 @@ More information on this initiative and how to contribute can be found [here](#f
...
+	- Create README.md file within each subfolder to track descriptions of the app, guidance on setting up the data collection, authors, versions, dates, and links to 3rd party docs
+- **Commit** the changes on your local machine to your own fork/branch
+- **Push** your work back up to your fork
+- **Submit** a pull request so that we can review your changes and publish
+
+
+### What are the guidelines for submitting my own content?
+
+Content can include vendor-based/custom applications, dashboards, search queries, field extraction rules (FERs), tooling, etc. Please complete each of the required fields in the [Sumo Logic Community Content Submission](https://forms.gle/KQBLBuMuUw85xtRi9) form, and follow the guidelines below:
+- Verify that all sensitive data has been removed BEFORE submitting content. This includes PII, Credentials, Network data, etc. If you provided screenshots, either blur or mark over this sensitive data.
```
Comment (kimsauce) [inline docs/integrations/community-ecosystem-apps.md:61]:
```suggestion
- Verify that all sensitive data has been removed or redacted BEFORE submitting content. This includes PII, Credentials, Network data, etc. If you provided screenshots, either blur or mark over this sensitive data.
```

File: docs/integrations/community-ecosystem-apps.md [hunk 3]
```diff
@@ -33,9 +33,43 @@ More information on this initiative and how to contribute can be found [here](#f
...
+
+Please follow the guidelines outlined in the [Sumo Logic Community Content Submission](https://forms.gle/KQBLBuMuUw85xtRi9) form if you would like to contribute your own content to this repository and have our team review it for publishing.
+
+**Alternative Submission:**
+Follow the "fork-and-pull" Git workflow:
+- **Fork** the repo on GitHub
+- **Clone** the project to your own machine
+- **Create** any new folders/subfolders necessary for your content
+	- Add your content to these folders
+	- Verify that all sensitive data has been removed BEFORE submitting a PR. This includes PII, Credentials, and Network data. Screenshots included (either blur or mark over).
```
Comment (kimsauce) [inline docs/integrations/community-ecosystem-apps.md:49]:
```suggestion
	- Verify that all sensitive data has been removed  or redacted BEFORE submitting a PR. This includes PII, Credentials, and Network data. Screenshots included (either blur or mark over).
```

**Comment by wjakelee** [comment]
closing this PR, I submitted a new one that is updated

**Pull request closed without merging**